plugin/copilot_chat.vim

```diff
@@ -18,7 +18,7 @@ command! -nargs=0 CopilotChatConfig call copilot_chat#config#view()
 command! -nargs=0 CopilotChatModels call copilot_chat#config#view_models()
 command! -nargs=0 CopilotChatSelectModel call copilot_chat#config#select_model()
 command! -nargs=? CopilotChatSave call copilot_chat#history#save(<q-args>)
-command! -nargs=? CopilotChatLoad call copilot_chat#history#load(<q-args>)
+command! -nargs=? -complete=customlist,copilot_chat#history#complete CopilotChatLoad call copilot_chat#history#load(<q-args>)
```

Comment:
this is a pretty cool trick! Did not know about this until reading this code. Like this a lot